src/coreclr/System.Private.CoreLib/src/System/RuntimeHandles.cs

```diff
@@ -650,12 +650,9 @@ internal static bool SatisfiesConstraints(RuntimeType paramType, RuntimeType[]?
             }
         }
 
-        [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern IntPtr _GetMetadataImport(RuntimeType type);
-
         internal static MetadataImport GetMetadataImport(RuntimeType type)
         {
-            return new MetadataImport(_GetMetadataImport(type), type);
+            return new MetadataImport(type.GetRuntimeModule(), type);
```

Comment:
This is slower than the existing code and it can trigger allocation of the RuntimeModule instance when there was no allocation before. Unclear whether it is problem in practice.